src/core/server/workspaces/saved_objects/workspace_saved_objects_client_wrapper.ts

```diff
@@ -96,6 +96,16 @@ export class WorkspaceSavedObjectsClientWrapper {
       objects: Array<SavedObjectsBulkCreateObject<T>>,
       options: SavedObjectsCreateOptions = {}
     ): Promise<SavedObjectsBulkResponse<T>> => {
+      if (options.workspaces) {
+        options.workspaces = options.workspaces.filter(
```

Comment:
Is that means the `bulkCreate` operation will be continued if user don't have permission to any one workspace? The `options.workspaces` will be empty, based current logic.